### PR TITLE
Add information on initial admin user / role

### PR DESCRIPTION
**Pull request by ben181231**
connect #258

### CODE REVIEW

File: content/server/configuring.md [hunk 1]
```diff
@@ -82,6 +82,16 @@ app:
     API_KEY: thisissupersecret
 ```
 
+<a name="administrator"></a>
+## Administrator
```
Comment (bensonby) [inline content/server/configuring.md]: will "Admin User" be a better title?

File: content/server/configuring.md [hunk 2]
```diff
@@ -82,6 +82,16 @@ app:
...
+
+After launching Skygear Server, an administrative role (`Admin`) and an
+adminitrator is created. The default username is `admin` and the default
```
Comment (bensonby) [inline content/server/configuring.md]: s/adminitrator/administrator

File: content/server/configuring.md [hunk 3]
```diff
@@ -82,6 +82,16 @@ app:
...
+password is `secret`.
+
+Please remember to change the password of the administrator before your
```
Comment (bensonby) [inline content/server/configuring.md]: Just curious, any recommended way of changing that? If there is, we can put it here too

**Comment by bensonby** [comment]
I think we also need to mention this from the "user" page of each SDK

File: content/server/configuring.md (hunk 1 above)
Comment (ben181231) [inline content/server/configuring.md]: Okay. I will update it.

File: content/server/configuring.md (hunk 3 above)
Comment (ben181231) [inline content/server/configuring.md]: @bensonby 
Should I list out links to all SDKs or just mention that changing password via SDKs?

Comment (bensonby) [inline content/server/configuring.md]: @ben181231 I am thinking about including sample code in each SDK's getting started to change admin password.. if there are no better ways to do it

File: content/server/configuring.md [hunk 4]
```diff
@@ -90,7 +90,39 @@ admin user is created. The default username is `admin` and the default
 password is `secret`.
 
 Please remember to change the password of the admin user before your
-application is push to production.
+application is push to production. The following snippet helps you to change
```
Comment (bensonby) [inline content/server/configuring.md]: @ben181231 better say it is a bash shell script

Comment (ben181231) [inline content/server/configuring.md]: Okay

**Pull request merged**